server/vertx/src/main/scala/sttp/tapir/server/vertx/VertxZioServerInterpreter.scala

```diff
@@ -75,14 +75,20 @@ trait VertxZioServerInterpreter[R] extends CommonServerInterpreter {
 
       rc.response.exceptionHandler { (t: Throwable) =>
         cancelRef.getAndSet(Some(Left(t))).collect { case Right(c) =>
-          c(FiberId.None)
+          rc.vertx().executeBlocking[Unit]((promise: Promise[Unit]) => {
```

Comment:
Thanks! Shouldn't we do the same in `VertxCatsServerInterpreter` ?

Reply:
It looks like this issue is not applied for `VertxCatsServerInterpreter` because `CancelToken` is function from `Unit` to `Future[Unit]`. So calling such function doesn't block caller and just returns `Future`.

Reply:
Ah ok:)